packages/styling/src/styles/theme.ts

```diff
@@ -89,7 +89,7 @@ function _makeSemanticColorsFromPalette(p: IPalette, isInverted: boolean): ISema
     focusBorder: p.black,
 
     errorBackground: !isInverted ? 'rgba(232, 17, 35, .2)' : 'rgba(232, 17, 35, .5)',
-    errorText: !isInverted ? p.redDark : '#ff5f5f',
+    errorText: !isInverted ? p.redDark : p.black,
```

Comment:
Hmmm. Is this right?

I would assume errorText color is the text color used for error text such as the errors when TextField has invalid content. Wouldn't this make the errors invisible on an inverted theme?